### ISSUE TITLE
Proposal: Lock API Quickstart

### ISSUE THREAD

**Issue by msfussell** [dapr/quickstarts#776]
## Describe the proposal
A quickstart for Locks API that use the Redis lock component to lock a row in a MySQL database that 2 applications are accessing in, when they are both receiving messages from a queue or pub/sub messages in a competing consumer pattern. This can build on the pub/sub or binding Quickstarts.

**Comment by paulyuk** [comment]
@msfussell would you be willing to build it?

**Comment by paulyuk** [comment]
I agree with proposal. We don't have horsepower to find it tho for now. Higher on my list is to fund kubernetes support. 

Do you have cycles ?

**Comment by kendallroden** [comment]
Will have someone work on for GHC
